PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -11,23 +11,27 @@ struct SourceData: Identifiable, Hashable {
     var id = UUID()
     var source: String
     var status: SourceValidation = .valid
+    var isEnabled: Bool
 
     enum SourceDataKeys: String, CodingKey {
         case source
+        case isEnabled
     }
 }
 
 extension SourceData: Encodable {
     func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: SourceDataKeys.self)
         try container.encode(source, forKey: .source)
+        try container.encode(isEnabled.description, forKey: .isEnabled)
     }
 }
 
 extension SourceData: Decodable {
     init(from decoder: Decoder) throws {
         let values = try decoder.container(keyedBy: SourceDataKeys.self)
         source = try values.decode(String.self, forKey: .source)
+        isEnabled = (try? values.decode(String.self, forKey: .isEnabled) == "true") ?? true
```

Comment:
This should use `.decodeIfPresent` as it provides backwards compatibility, and the decoding type should be `Bool.self` instead of `String.self` to avoid the string comparison (requires change above to be made).